internal/signer/darwin/keychain/keychain_test.go

```diff
@@ -48,7 +48,7 @@ func TestBytesToCFDataRoundTrip(t *testing.T) {
 }
 
 func TestEncrypt(t *testing.T) {
-	key, err := Cred("enterprise_v1_corp_client-signer-0-2018-07-03T10:55:10-07:00 K:1, 2:BXmhnePmGN4:0:18")
+	key, err := Cred("TestIssuer")
```

Comment:
Make "TestIssuer" a global string. Check if it should be `const`, I can't remember if Strings can be. Then use the Global variable in all the test cases.